src/get-started/create-service.md

```diff
@@ -209,54 +216,89 @@ For our cryptocurrency tutorial we need two transaction types:
 - Create a new wallet and add some money to it
 - Transfer money between two different wallets
 
-Declaration of any transaction should contain:
-
-- Service identifier
-- Unique (within the service) message identifier
-- Size of the fixed part of the message
-
-Exonum will use these constants for (de)serialization of the messages.
-
-### Creating New Wallet
-
-A Transaction to create a new wallet should contain the public key of the wallet
-and the name of the user who created this wallet:
+Service transactions are defined through `transactions!` macro
+that automatically assigns transaction IDs based on the declaration order:
```

Comment:
How do I know their values when I need to generate a transaction (and set a proper id) in an external system?

Reply:
You can count them manually or generate a rust-doc. 🙃 There is no better way for now.